core/src/main/java/org/apache/accumulo/core/client/ClientConfiguration.java

```diff
@@ -194,14 +196,17 @@ public static ClientConfiguration create() {
    *          the path to the configuration file
    * @since 1.9.0
    */
-  public static ClientConfiguration fromFile(File file) {
-    FileBasedConfigurationBuilder<PropertiesConfiguration> propsBuilder =
-        new FileBasedConfigurationBuilder<>(PropertiesConfiguration.class)
-            .configure(new Parameters().properties().setFile(file));
-    try {
-      return new ClientConfiguration(Collections.singletonList(propsBuilder.getConfiguration()));
+  public static ClientConfiguration fromFile(File file) throws FileNotFoundException {
+    var config = new PropertiesConfiguration();
+    try (var reader = new FileReader(file)) {
+      config.getLayout().load(config, reader);
+      return new ClientConfiguration(Collections.singletonList(config));
     } catch (ConfigurationException e) {
       throw new IllegalArgumentException("Bad configuration file: " + file, e);
+    } catch (FileNotFoundException fnfe) {
+      throw fnfe;
+    } catch (IOException e1) {
+      throw new UncheckedIOException("IOExcetion creating configuration", e1);
```

Comment:
It'd be better to add these exceptions to the existing `ConfigurationException` using multi-catch. That way, we don't introduce new exceptions in the public API method.